C/CMake/build_tarballs.jl

```diff
@@ -15,24 +15,23 @@ sources = [
 script = raw"""
 cd $WORKSPACE/srcdir/cmake-*/
 
-cmake -DCMAKE_INSTALL_PREFIX=$prefix
+cmake \
+    -DCMAKE_INSTALL_PREFIX=$prefix \
+    -DCMAKE_BUILD_TYPE:STRING=Release \
+    -DCMAKE_TOOLCHAIN_FILE=$CMAKE_TARGET_TOOLCHAIN
 
 make -j${nproc}
 make install
 """
 
-# These are the platforms we will build for by default, unless further
-# platforms are passed in on the command line.
+# Build for all supported platforms.
+#
 # CMake is in C++ and it exports the C++ string ABIs, but when compiling it with
 # the C++03 string ABI it seems to ignore our request, so let's just build for
 # the C++11 string ABI.
-platforms = [
-    Platform("i686", "linux"; libc="glibc", cxxstring_abi = "cxx11"),
-    Platform("x86_64", "linux"; libc="glibc", cxxstring_abi = "cxx11"),
-    Platform("x86_64", "linux"; libc="musl", cxxstring_abi = "cxx11"),
-]
-
-# platforms = expand_cxxstring_abis(platforms)
+platforms = filter(expand_cxxstring_abis(supported_platforms(; experimental=true))) do platform
+    !haskey(platform, "cxxstring_abi") || platform["cxxstring_abi"] == "cxx11"
+end
```

Comment:
Bit simpler :slightly_smiling_face: 
```suggestion
platforms = filter!(p -> cxxstring_abi(p) != "cxx03", expand_cxxstring_abis(supported_platforms()))
```